### ISSUE TITLE
WorldEdit plugin 3.7.9 not working with ArcLight Fabric 1.20.4

### ISSUE THREAD

**Issue by SirAussieman** [IzzelAliz/Arclight#1583]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-1.20.4-1.0.4-80ec5df

### OS & Java versions

OS: Unknown, Java: openjdk version "21.0.5" 2024-10-15

### Plugins and Mods

```raw
Mods:
• arclight 1.0.4+80ec5df
• BetterCompatibilityChecker-fabric-5.1.2+mc1.20.4.jar
• BetterPingDisplay-Fabric-1.20.4-1.1.1.jar
• BetterThirdPerson-Fabric-1.20-1.9.0.jar
• blastingraw-1.20.4-2-fabric.jar
• blastingsand-1.20.4-11-fabric.jar
• chat_heads-0.13.7-fabric-1.20.4.jar
• chatanimation-1.0.5.jar
• clayblasting-1.20.4-0-fabric.jar
• DiagonalFences-v20.4.1-1.20.4-Fabric.jar
• DiagonalWalls-v20.4.1-1.20.4-Fabric.jar
• DiagonalWindows-v20.4.1-1.20.4-Fabric.jar
• displaycase-fabric-1.20.4-1.0.jar
• fabric-api-0.97.2+1.20.4.jar
• ForgeConfigAPIPort-v20.4.3-1.20.4-Fabric.jar
• immersive-portals-5.1.7-mc1.20.4-fabric.jar
• minecraft 1.20.4
• PuzzlesLib-v20.4.52-1.20.4-Fabric.jar
• sodium-fabric-0.5.8+mc1.20.4.jar

Plugins (13): LuckPerms, SimpleRTP, Vault, WorldEdit, Essentials, EssentialsChat, AFKPlus, PlotSquared, Multiverse-Core, PocketGames, EssentialsDiscord, Multiverse-Inventories, Multiverse-Portals
```

### Description

Whenever I attempt to use the wand for WorldEdit, it tells me that it doesn't support my version of Bukkit and editing is unsupported. I realise that it would be better to use the Fabric version of the mod, but the WorldEdit plugin is a dependency for another plugin, PlotSquared. It is essential that I have PlotSquared on the server.

I have tried with all of my mods and plugins disabled (besides WorldEdit and PlotSquared), as well as a few minimal ones I'd like to have in the list above. No change.

The OS of the server is unknown due to it being hosted on Apex Hosting. I know it is using Java 21.0.5 though. The 'java -version' command does not work in Apex Hosting's console.

### Step to reproduce

1. Install WorldEdit plugin 7.3.9 (latest stable version for 1.20.4)
2. Load into the server
3. Attempt to use WorldEdit with //wand

### Logs

[latest.log](https://github.com/user-attachments/files/18057640/latest.log)

**Comment by qyl27** [comment]
The Fabric version of WorldEdit is suggested than Bukkit.

**Comment by SirAussieman** [comment]
I agree that I would like to use the Fabric version of WorldEdit, but it unfortunately crashes my server, even as the only mod that it tries to load.
[latest1.log](https://github.com/user-attachments/files/18058090/latest1.log)

Realised I forgot to add Fabric API to the mods folder, but no difference. Updated latest1.log file

**Comment by qyl27** [comment]
Ok, that is a bug.

**Comment by SirAussieman** [comment]
Preferrably, and if possible, I'd love to get the Bukkit plugin version working so it'd work with PlotSquared, as WorldEdit is a dependency.

Otherwise, if it's not possible, I'm happy to focus on the Fabric mod crashing the server as the primary bug instead.

**Comment by qyl27** [comment]
Currently, we are working on 1.21.1 and 1.21.3 porting and didn't have much time due to our real lives and jobs. It may need several weeks and I am sorry for that.  
These two issues should both be resolved.

**Comment by SirAussieman** [comment]
That's no problem at all. Happy to wait however long is needed while you guys work on 1.21.x. No rush!

**Comment by AgentMelinda** [comment]
Could you please try out the bukkit version 7.2.15
https://dev.bukkit.org/projects/worldedit/files/4586220
I hope it work, waiting for your replay.

**Comment by SirAussieman** [comment]
Exact same error with 7.2.15. Editing on unsupported versions is not available.

**Comment by MaxGilgen** [comment]
> Exact same error with 7.2.15. Editing on unsupported versions is not available.

Try the Mod version of WorldEdit. This works for me.

**Comment by SirAussieman** [comment]
> > Exact same error with 7.2.15. Editing on unsupported versions is not available.
> 
> Try the Mod version of WorldEdit. This works for me.

Mod version of WorldEdit 7.2.15 crashes the server for me. No additional mods, no plugins.
[latest.log](https://github.com/user-attachments/files/18109794/latest.log)

Or are you just referring to using the latest mod version of WorldEdit? Because, if so, check my latest1.log file a few comments up.

**Comment by MaxGilgen** [comment]
> > > Exact same error with 7.2.15. Editing on unsupported versions is not available.
> 
> > 
> 
> > Try the Mod version of WorldEdit. This works for me.
> 
> 
> 
> Mod version of WorldEdit 7.2.15 crashes the server for me. No additional mods, no plugins.
> 
> [latest.log](https://github.com/user-attachments/files/18109794/latest.log)
> 
> 
> 
> Or are you just referring to using the latest mod version of WorldEdit? Because, if so, check my latest1.log file a few comments up.

Oh i didn't read the whole discussion. Ehm jeah thats to bad. But i see you use the 1.20.4 i had the same problem there i use the 1.21.1 Snapshot of Arclight, there it's working fine for me, even MyWorlds Plugin does work properly. I downloaded the newest artifact form the newest build from last week.

**Comment by zdzym** [comment]
Add
`allow-editing-on-unsupported-versions: "I accept that I will receive no support with this flag enabled."`
in worldedit's `config.yml` solved this for me and I haven't met any problem so far.

**Comment by SirAussieman** [comment]
> Add `allow-editing-on-unsupported-versions: "I accept that I will receive no support with this flag enabled."` in worldedit's `config.yml` solved this for me and I haven't met any problem so far.

Wow, I didn't even know this existed. Thanks! This has allowed me to use the WorldEdit plugin.

I won't close this yet as, obviously, there's still a problem with the mod version of WorldEdit crashing the server, but at least the workaround for the plugin version has been found!

**Comment by qyl27** [comment]
> Add `allow-editing-on-unsupported-versions: "I accept that I will receive no support with this flag enabled."` in worldedit's `config.yml` solved this for me and I haven't met any problem so far.

Oh, thanks a lot! I will add it into the FAQ page in our wiki.

**Comment by vancevoj** [comment]
for me there is no default created config.yml for worldedit as a plugin. do i have to create this config file manually?

**Comment by SanGameplays** [comment]
> Add `allow-editing-on-unsupported-versions: "I accept that I will receive no support with this flag enabled."` in worldedit's `config.yml` solved this for me and I haven't met any problem so far.

You are the goat, GOTY, MvP, the ACE, the monster, thank you man :D
